examples/text-classification/run_pl.sh

```diff
@@ -23,7 +23,7 @@ mkdir -p $OUTPUT_DIR
 # Add parent directory to python path to access lightning_base.py
 export PYTHONPATH="../":"${PYTHONPATH}"
 
-python3 run_pl_glue.py --data_dir $DATA_DIR \
+python3 run_pl_glue.py --gpus 2 --data_dir $DATA_DIR \
```

Comment:
should 2 gpus be the default?

Reply:
it should not.

Reply:
So should I add environment variables in the shell script just like $DATA_DIR etc.?

Reply:
DATA_DIR is solid. My only objection is gpus=2.